pulpcore/cli/common/generic.py

```diff
@@ -865,7 +865,6 @@ def content_modify(
         repo_ctx.modify(repo_ctx.pulp_href, ac, rc, base_version.pulp_href)
 
     command_decorators = {
```

Comment:
```suggestion
    command_decorators: Dict[click.Command, Optional[list[_FC]]] = {
```

Reply:
Sorry the MetaType is `List[...]` (capital `L`).

Reply:
Easy peasy and straighforward:
```python
    command_decorators: Dict[click.Command, Optional[List[Callable[[_FC], _FC]]]] = {
```

Reply:
Right. That's the one we need.

Reply:
Do we see `Callable[[_FC], _FC]` elsewhere? Then we should put a name on it.

Reply:
Yep, we do. Bunch of times:
```
[lmjachky@localhost pulp-cli]$ git grep "_FC"
pulpcore/cli/common/generic.py:_FC = TypeVar("_FC", Callable[..., Any], click.Command)
pulpcore/cli/common/generic.py:def pulp_option(*args: Any, **kwargs: Any) -> Callable[[_FC], _FC]:
pulpcore/cli/common/generic.py:def resource_option(*args: Any, **kwargs: Any) -> Callable[[_FC], _FC]:
pulpcore/cli/common/generic.py:    command_decorators: Dict[click.Command, Optional[List[Callable[[_FC], _FC]]]] = {
```

I am adding a constant for that.